pgcontents/tests/test_synchronization.py

```diff
@@ -221,20 +225,20 @@ def test_generate_files(self):
                     'test_generate_files2']
         (managers, paths) = self.populate_users(user_ids)
 
-        def get_file_dt(user_id, idx):
-            path = paths[user_id][idx]
+        def get_file_dt(idx):
+            (user_id, path) = paths[idx]
             return managers[user_id].get(path, content=False)['last_modified']
 
-        # Find a split datetime midway through each user's list of files
-        split_idx = len(paths[user_ids[0]]) // 2
-        split_dts = [get_file_dt(user_id, split_idx) for user_id in user_ids]
+        # Find three split datetimes
+        split_idxs = [i * (len(paths) // 4) for i in range(1, 4)]
```

Comment:
The purpose of this is so that we get 3 indexes within `paths`. `range(1, 4)` and the `// 4` together give us 1/4, 2/4, and 3/4 of the way through the list.

We could make this depend on the number 3, for instance:
```
n = 3
split_idxs = [i * (len(paths) // (n + 1)) for i in range(1, n + 1)]
```